app/src/main/java/com/sapuseven/untis/activities/SettingsActivity.kt

```diff
@@ -73,7 +75,7 @@ class SettingsActivity : BaseComposeActivity() {
 		private const val URL_WIKI_PROXY = "$URL_GITHUB_REPOSITORY/wiki/Proxy"
 	}
 
-	@OptIn(ExperimentalMaterial3Api::class)
+	@OptIn(ExperimentalMaterial3Api::class, ExperimentalMaterialApi::class)
```

Comment:
Don't think that's needed - MaterialApi shouldn't be combined with Material3Api
```suggestion
	@OptIn(ExperimentalMaterial3Api::class)
```